lib/internal/bootstrap_node.js

```diff
@@ -190,6 +190,12 @@
   }
 
   function setupGlobalVariables() {
+    Object.defineProperty(global, 'global', {
```

Comment:
By the time this code is run, the `global` property of the global object is already defined, and this line redefines it to be non-enumerable. This works, but I'm wondering if there's a less roundabout way to do it. (I assume it would be related to [this](https://github.com/nodejs/node/blob/a1652324cd8cc77a1964286fa49900d89755ab0d/src/node.cc#L3429), but I'm not sure about the specifics of how to make a property non-enumerable from the C++ side.)